test/integration/e2e.spec.ts

```diff
@@ -1387,4 +1398,156 @@ describe('e2e', function () {
       }
     });
   });
+
+  describe('FeiRari Tribe Staking Rewards', async () => {
+    let tribe: Contract;
+    let tribalChief: Contract;
+    let timelockAddress: string;
+    let tribePerBlock: BigNumber;
+    let autoRewardsDistributor: Contract;
+    let rewardsDistributorAdmin: Contract;
+    let stakingTokenWrapper: Contract;
+    let rewardsDistributorDelegator: Contract;
+    const poolAllocPoints = 1000;
+    const pid = 3;
+    let optimisticTimelock: SignerWithAddress;
+    let totalAllocPoint: BigNumber;
+
+    before(async () => {
+      stakingTokenWrapper = contracts.stakingTokenWrapperRari;
+      rewardsDistributorDelegator = contracts.rariRewardsDistributorDelegator;
+      tribePerBlock = toBN('75').mul(toBN(e18));
+      tribalChief = contracts.tribalChief;
+      rewardsDistributorAdmin = contracts.rewardsDistributorAdmin;
+      autoRewardsDistributor = contracts.autoRewardsDistributor;
+      tribe = contracts.tribe;
+
+      optimisticTimelock = await ethers.getSigner(contracts.optimisticTimelock.address);
+      await hre.network.provider.request({
+        method: 'hardhat_impersonateAccount',
+        params: [optimisticTimelock.address]
+      });
+      await forceEth(optimisticTimelock.address);
+    });
+
+    describe('Staking Token Wrapper', async () => {
+      it('init staking token wrapper', async function () {
+        totalAllocPoint = await tribalChief.totalAllocPoint();
+        expect(stakingTokenWrapper.address).to.be.equal(await tribalChief.stakedToken(3));
+        expect((await tribalChief.poolInfo(pid)).allocPoint).to.be.bignumber.equal(toBN(poolAllocPoints));
+        expect(totalAllocPoint).to.be.equal(toBN(3100));
```

Comment:
This test was designed to call the init method, but since that has already been done by a separate deployment script, I removed that action in this test.